C++/Navio/Common/gpio.h

```diff
@@ -47,6 +47,7 @@ class Pin {
     void    toggle();
 
 private:
+    int getPeriBase() const;
```

Comment:
What's with return type? Did you compile it before push?

Reply:
Thank you! I miss that while committing.